backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/QueueManagementTest.java

```diff
@@ -103,32 +100,6 @@ void updateItemInQueue() throws Exception {
     assertEquals(dateTested, singleQueueEntry.path("dateTested").asText());
   }
 
-  @Test
-  void twoSimultaneousUpdates() throws Exception {
```

Comment:
as an aside, I don't think that specifying errors in `runQuery` works as I thought it did, and this unit test wasn't working as intended to begin with